### ISSUE TITLE
train new model via MegaFace is low accuracy

### ISSUE THREAD

**Issue by axmadjon** [davidsandberg/facenet#671]
hi @davidsandberg 
I'm trying to train a new model with and without pre-train model
but during the training of skills poorly trains
about 100> epochs do not rise 15-20 percent accuracy on LFW
but I'm training a new model from the base MegaFace

if __name__ == '__main__':
    train_softmax(
        # i_pretrained_model='', # i_pretrained_model='/home/smartup/facenet/pretrained/model-20180207-130233.ckpt-30000',
        i_pretrained_model='D:/new_face_dataset/model/model-20170512-110547.ckpt-250000',
        i_logs_base_dir='D:/new_face_dataset/facenet/logs',
        i_models_base_dir='D:/new_face_dataset/facenet/model',
        i_data_dir='C:/facenet/',
        i_image_size=160,
        i_model_def='cv.tensor.facenet.models.inception_resnet_v1',
        i_optimizer='RMSPROP',
        i_max_nrof_epochs=200,
        i_gpu_memory_fraction=0.5,
        i_keep_probability=0.8,
        i_weight_decay=2e-4,
        i_center_loss_factor=5e-5,
        i_center_loss_alfa=0.9,
        i_lfw_pairs='D:/DATASET/dataset/vis-umass.edu/pairsDevTrain.txt',
        i_lfw_dir='D:/DATASET/dataset/vis-umass.edu/lfw',

        i_learning_rate=-1,
        i_learning_rate_schedule_file='D:/new_face_dataset/face_dataset/learnning_rate.txt')

please help where I'm doing wrong?

**Comment by axmadjon** [comment]
The first test without training gave

 Runnning forward pass on LFW images
 Accuracy: 0.905+-0.016
 Validation rate: 0.28591+-0.28602 @ FAR=0.00045

such a result, and after it carefully decreased

 Runnning forward pass on LFW images
 Accuracy: 0.357+-0.113
 Validation rate: 0.00409+-0.00591 @ FAR=0.00045

**Comment by tengerye** [comment]
I did the similar thing on my own dataset and the accuracy falled to 0.5 in the end.
